tests/test_rpc.py

```diff
@@ -60,7 +60,7 @@ def test_rpc_errors(arb_addr, to_call, testdir):
     if exposed_mods == ['tmp_mod']:
         # create an importable module with a bad import
         testdir.syspathinsert()
-        # module should cause raise a ModuleNotFoundError at import
+        # module should cause a raise of a ModuleNotFoundError at import
```

Comment:
Warning, I'm pedantic. I'd have worded this as `module should raise a ModuleNotFoundError at import`. Shorter and grammatically correct.